pkg/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -430,6 +430,16 @@ type NetworkConfig struct {
 	//     Used to statically set the nameservers for the host.
 	//     Defaults to `1.1.1.1` and `8.8.8.8`
 	NameServers []string `yaml:"nameservers,omitempty"`
+	//   description: |
+	//     Allows for extra entries to be added to /etc/hosts file
+	//   examples:
+	//     - |
+	//       extraHostEntries:
+	//         - ip: 192.168.1.100
+	//           aliases:
+	//             - test
+	//             - test.wtf.bbq
```

Comment:
We probably shouldn't have `test.wtf.bbq` in here. Silly, I know, but I think we should keep examples free of anything that could be offensive, and use very neutral and generic things like `example.com`, `example`, `domain.tld`, `domain`, etc.

Reply:
fixed

Reply:
👍 Thank you.